changelog_page.php

```diff
@@ -241,7 +241,7 @@ function print_project_header_changelog( $p_project_name ) {
 	$t_resolved = config_get( 'bug_resolved_status_threshold' );
 
 	# grab version info for later use
-	$t_version_rows = version_get_all_rows( $t_project_id, null, false );
+	$t_version_rows = version_get_all_rows( $t_project_id, /* released */ null, /* obsolete */ false );
```

Comment:
@dregad I disagree that this is code clutter.  I think it is the a good way to provide annotation of parameters where it is not clear.  My other alternative would be to assign these values to variables and then use them.

@atrol I also thought about the use of constants, but I think it will cause too many constants and will have the risk of being re-used in a context where the value doesn't match the target function expectation, hence, I would rather use defines for regular constants and enums, rather than as a way to provide a friendly name to values that are null, true, and false.

Having said that, I have used @dregad suggest for this one.  But we should revisit this.